src/unity/lib/visualization/histogram.cpp

```diff
@@ -188,11 +188,21 @@ void histogram_result::add_element_simple(const flexible_type& value) {
    * add element to histogram
    */
 
-  // ignore undefined values
+
   if (value.get_type() == flex_type_enum::UNDEFINED) {
     return;
   }
 
+  if (value.get_type() == flex_type_enum::FLOAT &&
+      std::isinf(value.get<flex_float>())) {
```

Comment:
Might want to use [isfinite](http://en.cppreference.com/w/cpp/numeric/math/isfinite) to cover NaN as well.

Reply:
Updated